### PR TITLE
test(circile ci automation ):setting up code climate

### PR DESCRIPTION
**Pull request by Mukunzijames**
issue in referencing?

Questions:

What does this PR do?

working on circleci and codeclimate for mantaining code

- i was setting up circleci for automating code and keep it clean
- setting up codeclimate for automation and mantaining code

How should this be manually tested?
-run "npm run dev"
-run "npm run test"

Any background context you want to provide?

-dummy data are provided before having the real data

What are the relevant pivotal tracker/Trello stories?

id of the project
#187300136
#187300134
#187300135

### CODE REVIEW

File: test/server-run.ts [hunk 1]
```diff
@@ -17,4 +21,88 @@ describe('API Tests', () => {
...
+    process.env.PROD_DATABASE_PORT = '';
+    process.env.DATABASE_NAME = '';
+    process.env.DATABASE_USER = '';
+    process.env.DATABASE_PASSWORD = '';
+    process.env.DATABASE_HOST = '';
+    process.env.DATABASE_PORT = '';
+    process.env.DIALECT = '';
+  });
+
+  it('should return database configuration for development environment', () => {
```
Comment (fab-ryan) [inline test/server-run.ts]: why testing configuration  while should not be tested

**Review by fab-ryan — CHANGES_REQUESTED** [review]
things we talked nothing worked on it script for testing and no rebase happened

**Review by Ruthuwamahoro — APPROVED** [review]
Good job

**Review by IRASUBIZA-Elyse — APPROVED** [review]
good.

**Review by RWEMAREMY — APPROVED** [review]
Good work

**Review by fab-ryan — APPROVED** [review]

**Pull request merged**